src/app/clients/getClientsTableData.ts

```diff
@@ -1,20 +1,25 @@
 import supabase from "@/supabaseClient";
 import { Schema } from "@/database_utils";
-import { Datum } from "@/components/Tables/Table";
 
-export interface ClientsTableRow extends Datum {
-    primaryKey: string;
+export interface ClientsTableRow {
     parcelId: Schema["parcels"]["primary_key"];
-    flaggedForAttention: boolean;
-    requiresFollowUpPhoneCall: boolean;
+    primaryKey: Schema["clients"]["primary_key"];
     fullName: Schema["clients"]["full_name"];
     familyCategory: string;
     addressPostcode: Schema["clients"]["address_postcode"];
-    collectionCentre: string;
-    congestionChargeApplies: boolean;
+    deliveryCollection: {
+        collectionCentre: string;
+        congestionChargeApplies: boolean;
+    };
     packingTimeLabel: string;
```

Comment:
Would it be helpful to make it more specific so it can only be like a valid time string rather than random gibberish like "!123456789876543e4565"